db/migrate/20190508011342_create_cheatsheets.rb

```diff
@@ -0,0 +1,13 @@
+class CreateCheatsheets < ActiveRecord::Migration[5.2]
+  def change
+    create_table :cheatsheets do |t|
+      t.string :title
```

Comment:
You *may* want to do something like this ...
```ruby
, :limit => 1000
```

on your migrations which will limit the size of the strings in the database. Probably not necessary here, but something to know about anyway.

Reply:
That's a good idea!! Don't want people getting carried away with the titles lol